docs/book/features/orphan_volumes.md

```diff
@@ -0,0 +1,50 @@
+## Orphan volumes detection and clean-up
+Orphan volumes are vSphere volumes that are present on a vSphere datastore but there is no corresponding PersistentVolume in Kubernetes clusters on the vCenter.
+
+Orphan volumes are often created when CNS solution creates more than one FCD for a Persistent Volume in the Kubernetes cluster. This could occur when the vCenter components are slow, storage is slow, vCenter service restarts, connectivity issues between vCenter and ESXi hosts etc. Since these orphan volumes occupy space in the datastore and are not really used in Kubernetes, it's useful to identify and cleanup orphan volumes periodically.
+
+This functionality provides a set of APIs to detect and delete orphan volumes on-demand, and also provide an option to turn on automatic deletion periodically.
+
+### What qualifies as an orphan volume ?
+A volume qualifies as an orphan volume if it meets all of the below conditions:
+
+1) A PersistentVolume in any of the registered kubernetes clusters is not using the volume.
+2) Volume was dynamically provisioned using vSphere CSI driver.
+3) Volume exists for more than 50 minutes(or `orphan-volume-detection-interval-mins` value configured in `cnsmanager-config` configmap) at the time of orphan volumes detection.
```

Comment:
Do you think we should capture the information that the worst case time for a volume to be detected as an orphan by the tool is twice `orphan-volume-detection-interval-mins` (or 2 cache update cycles).

Reply:
I think we should abstract implementation level details in the doc. Btw why do you think it will be 2 cache update cycles?

Reply:
The worst-case scenario for detecting a volume as an orphan by cns-manager is the 2 cycle scenario. 

For example, when the cache update duration is set to the default value of 50 minutes and a cache update job finishes at t1 minutes, and a vStorage object is created for the volume just after t1 (say t1+c), the next cache update job would run at t1+50. However, the cache update job won't qualify the volume as an orphan because the difference between the current time and the vStorageObject create time (t1+50-t1-c) is less than 50 minutes. Therefore, it would require another cache update cycle to detect this volume as an orphan.

Reply:
I agree that we should abstract the implementation details but I think it makes sense to let the user know what to expect when invoking the LIST OV endpoint.

Reply:
Okay I have added these details.